docs/docs/doc-de-visao.md

```diff
@@ -0,0 +1,207 @@
+
+### 1.5 Visão Geral
+
+A organização do documento é feita de maneira a prover ao leitor a capacidade de através do mesmo entender o produto em seus vários aspectos de forma coesa. Para tal, são apresentados primeiramente os tópicos referentes a função geral do software e as motivações que levaram a sua criação, após isso, é descrito o posicionamento do produto em relação ao mercado e as partes interessadas, incluindo a forma como a criação do sistema afetará os usuários. Por fim, são descritas as principais funcionalidades do software, bem como algumas de suas restrições e requisitos.
+
+***
+## 2. Posicionamento
+***
+
+### 2.1 Oportunidade de Negócios
+
+O ComexStat, tornando mais fácil o acesso aos dados relacionados ao comércio exterior de bens, e posteriormente serviços, no Brasil, proporciona também aos usuários a possibilidade de trabalhar com as informações desejadas de forma mais adequada as suas necessidades. Resolvendo portanto principalmente os problemas relacionados a atual dificuldade em acessar os dados e tirar deles alguma conclusão, por conta da forma pela qual os mesmos são disponibilizados e por seu grande volume, além da falta de ferramentas que possibilitem ao usuário filtrar, visualizar graficamente, e exportar de formas variadas esses dados, recursos estes que são presentes no ComexStat. Dessa forma, o sistema dá mais eficiência para as funções do MDIC quanto a exposição das informações, bem como para qualquer gestor ou produtor de bens ou serviços que tenha interesse nos dados disponibilizados para fazer qualquer tipo de análise.
+
+### 2.2 Instrução do Problema
+
+|  | |
+| :--------: | :-------: |
+| **Problema**| Dados relacionados ao comércio exterior de bens e serviços no Brasil são abertos a população porém de difícil uso e interpretação, por conta do seu grande volume e formato disponibilizado|
+| **Funções afetadas** | Análise e visualização dos dados de comércio de bens e serviços |
+| **Efeito** | Os dados acabam por não serem utilizados|
+| **Solução** |Criação de uma plataforma web que dá ao usuário a possibilidade de acessar, filtrar e visualizar os dados de forma mais intuitiva|
+
+### 2.3 Instrução de Posição do produto
+
+|  |  |
+| :--: | :--: |
+| **Público Alvo** | Gestores e produtores de bens ou serviços|
+| **Carência** | Necessidade de um melhor acesso e ferramentas de manejo a informações relacionadas ao comércio exterior de bens e serviço no Brasil |
+| **Solução** | ComexStat |
+| **Descrição da Solução** | Uma plataforma web que permita ao usuário o acesso aos dados de forma fácil e intuitiva, a partir de uso de filtros, detalhamento e ferramentas de visualização gráfica, além da possibilidade de exportação das informações em formatos como PDF e .csv  |
+| **Diferenciais** | Facilidade no acesso e ferramental para proporcionar uma melhor interpretação dos dados |
+
+***
+## 3. Descrição dos Envolvidos e dos Usuários
+***
+
+### 3.1. Resumo dos Envolvidos
```

Comment:
Acho que não ficou tão bem delimitado o que cada papel faz, deem uma olhada nesse documento aqui:

https://fga-eps-mds.github.io/2018.2-ComexStat/docs/processo#2-papeis

Olhando os papéis, será que não fica muito raso colocar dessa forma? Parece que MDS só desenvolve e EPS só observa, quando na verdade todos nós estamos inseridos em um ambiente que utiliza metodologias ágeis para entregar um produto de qualidade.

Reply:
Feito!